### PR TITLE
Feature: Updated scroll to top button

### LINKED ISSUE #164: [Feature Request] Make The Scroll to top button a little smaller and transparent.
### Description

Currently, the 'scroll to top' button occupies a considerable amount of space and overlaps BID images, especially on mobile devices it looks a bit off. Reducing its size and making it more transparent would give it a sleeker appearance and improve the user interface.

### Screenshots

Now
image

After some small changes
image

### Checklist

- [X] I have checked for duplicate [issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
- [X] I have read the [Contribution Guidelines]((https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md))
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by MeetThakur**
## Proposed Changes and Benefits

Reduced the size and made the scroll-to-top button transparent to enhance the UI.
also updated my username in contributors.json and CONTRIBUTORS.md

## Screenshots
before:
image
after:
image

### Checklist

- [✔ ] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [✔  ] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [✔  ] I have listed my proposed changes and their benefits.
- [ ✔ ] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

File: CONTRIBUTORS.md
Comment (XanderRubio) [inline CONTRIBUTORS.md:1]: Makes sense to go with your name for marketing purposes👏🏻

**Review by XanderRubio — APPROVED** [review]
Hello @MeetThakur,

I have reviewed the UI enhanced scroll and it appears much better with transparency and smoother scrolling. Additionally, great job on shrinking the size to save real estate space for contributor images and the button still looks great when the screen size is reduced. Thank you for your initiative to assist and improve, it is greatly appreciated. Keep up the great work and have a wonderful day!

**Comment by XanderRubio** [comment]
@MeetThakur Congratulations for your pull request being merged, and this successfully closes issue #164 ✅

**Comment by MeetThakur** [comment]
@XanderRubio Thank you very much, glad I could help. Looking forward contributing more as much I can.
Have a great day.

**Comment by XanderRubio** [comment]
> @XanderRubio Thank you very much, glad I could help. Looking forward contributing more as much I can. Have a great day.

Thank you, @MeetThakur, and we are grateful for your enthusiasm for the Before I Die code. You also have a nice day!

**Pull request merged**